lang/en.json

```diff
@@ -936,32 +936,56 @@
     "ActivityUses": {
       "Label": "Activity Uses",
       "Prompt": "Consume Activity Use?",
+      "PromptHintDecrease": "Spend <strong>{cost}</strong> {use} of this activity.",
+      "PromptHintIncrease": "Recover <strong>{cost}</strong> {use} of this activity.",
       "Warning": "uses on {item}'s activity {activity}"
     },
     "Attribute": {
       "Label": "Attribute",
       "Prompt": "Consume Attribute?",
+      "PromptHintDecrease": "Decrease <code>{attribute}</code> by <strong>{cost}</strong>.",
+      "PromptHintIncrease": "Increase <code>{attribute}</code> by <strong>{cost}</strong>.",
       "Warning": "{attribute} amount"
     },
     "HitDice": {
       "Label": "Hit Dice",
       "Prompt": "Consume Hit Die?",
+      "PromptHintDecrease": "Spend <strong>{cost}</strong> {denomination} {die}.",
+      "PromptHintIncrease": "Recover <strong>{cost}</strong> {denomination} {die}.",
       "Warning": "{denomination} hit dice"
     },
+    "HitDie": {
+      "one": "hit die",
+      "other": "hit dice"
+    },
     "ItemUses": {
       "Label": "Item Uses",
       "Prompt": "Consume Item Use?",
+      "PromptHintDecrease": "Spend <strong>{cost}</strong> {use} of {item}.",
+      "PromptHintIncrease": "Recover <strong>{cost}</strong> {use} of {item}.",
       "Warning": "uses on {name}"
     },
     "Material": {
       "Label": "Material",
       "Prompt": "Consume Material?",
+      "PromptHintDecrease": "Reduce quantity of {item} by <strong>{cost}</strong>.",
+      "PromptHintIncrease": "Increase quantity of {item} by <strong>{cost}</strong>.",
       "Warning": "of {name}"
     },
+    "SpellSlot": {
+      "one": "{level} slot",
+      "other": "{level} slots"
+    },
     "SpellSlots": {
       "Label": "Spell Slots",
       "Prompt": "Consume Spell Slot?",
```

Comment:
Might be nice if the labels cost the same treatment as the hints, i.e. `"Restore Spell Slot?"`, etc.